app/assets/stylesheets/govuk-component/_breadcrumbs.scss

```diff
@@ -34,3 +34,12 @@
     }
   }
 }
+
+.breadcrumb-for-current-page--inverse {
```

Comment:
We don't need this any more as the class isn't being used anywhere

Reply:
Thanks! Amended!